### PR TITLE
SC-3278, SC-3279, SC-3280 - datasource overview & new page

### PR DESCRIPTION
**Pull request by CeEv**
# Issue: SC-3279, SC-3280

https://ticketsystem.schul-cloud.org/browse/SC-3279
https://ticketsystem.schul-cloud.org/browse/SC-3280

Depends on: https://github.com/schul-cloud/nuxt-client/pull/502

## Description

- Follow Up Tickets are already created:
  - https://ticketsystem.schul-cloud.org/browse/SC-3284
  - https://ticketsystem.schul-cloud.org/browse/SC-3199

## Screenshot

image
image

### CODE REVIEW

File: src/pages/administration/datasources/index.vue [hunk 1]
```diff
@@ -3,12 +3,37 @@
...
 			<base-breadcrumb :inputs="inputs" />
 			<h3>Datenquellen</h3>
-			<empty-state :image="imgsrc">
-				<template v-slot:description
-					>Noch keine Datenquellen vorhanden. Mit dem Plus unten rechts kannst
-					du eine Datenquelle hinzufügen.
-				</template>
-			</empty-state>
+
+			<section v-if="!datasources || datasources.length === 0" class="section">
```
Comment (adrianjost) [inline src/pages/administration/datasources/index.vue]: Use article instead of section

https://www.smashingmagazine.com/2020/01/html5-article-section/

**Comment by adrianjost** [comment]
@dominikbruechner should be fixed now

File: src/components/molecules/DatasourceCard.stories.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { storiesOf } from "@storybook/vue";
 import { text } from "@storybook/addon-knobs";
 
 import DatasourceCard from "./DatasourceCard";
-import ExampleImage from "@assets/img/logo/logo-webuntis.svg";
+import ExampleImage from "@assets/img/datasources/logo-webuntis.png";
```
Comment (dominikbruechner) [inline src/components/molecules/DatasourceCard.stories.js]: does it need to be png? is it compressed? does it work well with zooming in?

Comment (adrianjost) [reply src/components/molecules/DatasourceCard.stories.js]: we do not have a svg file for that so we have no choice at the moment.

**Pull request merged**